bats/tests/helpers/utils.bash

```diff
@@ -307,6 +307,32 @@ EOF
     fi
 }
 
+# create_file path/to/file <<< "contents"
+# Create a new file with the provided path; the contents of standard input will
+# be written to that file.  Analogous to `cat >$1`.  Will create any parent
+# directories.
+create_file() {
+    # On Windows, avoid creating files from within WSL; this leads to issues
+    # where the WSL view of the filesystem is desynchronized from the Windows
+    # view, so we end up having ghost files that can't be deleted from Windows.
+    if ! is_windows; then
+        mkdir -p "$(dirname "$1")"
+        cat >"$1"
+        return
+    fi
+
+    local contents # Base64 encoded file contents
+    contents="$(base64)" || return
```

Comment:
All the `|| return` clauses are not really necessary, as it seems unlikely that anyone would ever run it via `run` or `try`.

Right now it is only called by `start_container_engine`, which is also not likely to be called via `run` or `try`.